buildSrc/src/main/groovy/com/google/firebase/gradle/plugins/license/LicenseResolverPlugin.groovy

```diff
@@ -52,7 +51,6 @@ class LicenseResolverPlugin implements Plugin<Project> {
             [new AndroidSdkTermsFetcher(),
              new Apache2LicenseFetcher(),
              new AnotherApache2LicenseFetcher(),
```

Comment:
What about this one? should we remove it too?

Reply:
It comes from a site that requires a different method to extract the license text, so the fetchers cannot be merged together.